app/scripts/directives/serviceBinding.js

```diff
@@ -17,22 +20,62 @@
     templateUrl: 'views/directives/_service-binding.html'
   });
 
-  function ServiceBinding() {
+  function ServiceBinding(APIService, DataService, SecretsService) {
     var ctrl = this;
+    ctrl.showParameterValues = false;
+
+    var context = {
+      namespace: ctrl.namespace
+    };
+
+    var updateParameterData = function() {
+      ctrl.parameterData = angular.copy(_.get(ctrl.binding, 'spec.parameters', {}));
+
+      _.each(_.get(ctrl.binding, 'spec.parametersFrom'), function(parametersSource) {
+        DataService.get("secrets", _.get(parametersSource, 'secretKeyRef.name'), context).then(function(secret) {
```

Comment:
Need to check if this user can get secrets. Project viewers can't, but they will be able to see the instance.